.github/workflows/image-builder.yml

```diff
@@ -0,0 +1,37 @@
+name: Build and publish the container image
+
+on:
+  push:
+    branches:
+      - "main"
+  workflow_dispatch:
+
+jobs:
+  build_dockerfile:
+    runs-on: ubuntu-20.04
+    env:
+      IMAGE_NAME: ansible-community/ansible-website
+      TAGS: latest
+      REGISTRY: ghcr.io
+      REGISTRY_USER: ${{ github.actor }}
```

Comment:
I'd probably put this on the global level, rather than in a job to reduce nesting.

Reply:
good idea. thanks.